src/questmaker/EditQuestDialogCanvas.java

```diff
@@ -133,7 +133,7 @@ public void mousePressed(MouseEvent e) {
                 DecisionEditDialog ded = new DecisionEditDialog(null, tempDecision);
                 this.repaint();
             } else if (an != null) {
-                RequestTextDialog rtg = new RequestTextDialog(null, 300, 200);
+                RequestTextDialogAnswer rtg = new RequestTextDialogAnswer(null, an.popis, 300, 200);

```

Comment:
Toto je podla mňa overkill, pretože popis by mal byť čisto jedno dve slová aby si charakterizoval danú odpoveď, napr odpoveď: "Teraz sú ťažké časy. Nikto nevie čo bude, kam pojdeme. Každý deň je pre mňa novou výzvou. Vidím však, že potrebujete pomoc. Zoberte si aspoň tento kus chleba, aspoň Vám to pomože" Može byť charakterizovaná popisom: "Dať chleba", netreba posielať ten popis do RequestTextDialogAnswer a zbytočne tak vytvárať celú classu novú